src/installing-workflow/configuring-object-storage.md

```diff
@@ -211,6 +141,8 @@ If the `DATABASE_STORAGE` backend is configured as "s3", the database will recei
 
 ### Google Cloud Storage (Interoperability Mode)
 
+[Google Cloud Storage](https://cloud.google.com/storage/) (GCS) can interoperate with the S3 API using a feature called [interoperability](https://cloud.google.com/storage/docs/interoperability). If you choose to use GCS for object storage for database, you'll have to turn on this interoperability mode. In order to do so, please follow the steps in the [GCS migration documentation](https://cloud.google.com/storage/docs/migrating?hl=en_US#migration-simple).
```

Comment:
Currently i haven't made any changes for the postgres, hence i had moved this from the generic section to database section.